content/secrets/12-walkthrough-step-09/copilot.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+
+Create the secure parameter:
+
+```bash
+aws ssm put-parameter --name GH_WEBHOOK_SECRET --value secretvalue1234 --type SecureString --tags Key=copilot-environment,Value=test Key=copilot-application,Value=ecsworkshop
+```
+Note that the secure parameter is tagged with the copilot application name.
+
+Modify the `copilot\todo-app\manifest.yml` and uncomment the `secrets` section:
```

Comment:
Can you do [heredoc](https://linuxize.com/post/bash-heredoc/#basic-heredoc-examples) and cat this into the manifest instead?

Reply:
Turns out this section isnt even needed.